node-status-monitor-client/src/main/java/com/sequenceiq/node/health/client/CdpNodeStatusMonitorClient.java

```diff
@@ -63,7 +63,11 @@ public void close() {
 
     public CdpNodeStatuses nodeStatusReport(CdpNodeStatusRequest request) throws CdpNodeStatusMonitorClientException {
         CdpNodeStatuses.Builder responseBuilder = CdpNodeStatuses.Builder.builder()
-                .withNetworkReport(nodeNetworkReport(true, request.isSkipObjectMapping()))
+                .withNetworkReport(nodeNetworkReport(true, request.isSkipObjectMapping()));
+        if (request.isNetworkOnly()) {
+            return responseBuilder.build();
+        }
+        responseBuilder
```

Comment:
Please put the rest of the function in an else branch

Reply:
I add the else branch.